tools/diagnostics/full_bundle.py

```diff
@@ -146,6 +150,11 @@ def create(self) -> (int, "FullBundle"):
             for t in active_marathon_service.get("tasks", [])
             if is_service_scheduler_task(self.package_name, self.service_name, t)
         ]
+        if len(scheduler_tasks) == 0:
+            log.warn(
+                "Could not find scheduler tasks for '%s'. Please check the 'dcos_services.json' file.",
```

Comment:
I see a few issues here:
1. please make `dcos_services.json` a constant and use it both here and on line 121. Such reference will ensure that the warning will not rot if someone changes the filename.
2. the warning is not very precise, we should explain that we're looking among active tasks of marathon service with id such-and-such
3. it's not terribly clear what the end-user is supposed to look for in that file? Look at other marathon services? unreachable or completed tasks too?
4. the json file tends to be large and very hard to view without a viewer which can collapse subtrees, even if someone knows what structure to expect. Perhaps we can do this search for them? Although this is probably outside of scope of this PR.

Reply:
I agree with `4`, but I'll leave it out of the scope of this PR.

I think I addressed the other points here: 5bd6e17e976286677a13aad18b6f56a26dc55895